### PR TITLE
S5 thebestonatrack

### PR DESCRIPTION
**Pull request by mgallagher56**

### CODE REVIEW

**Comment by LukeChaston** [comment]
So. The syntax and everything looks ok but at the end I'm only getting an empty array.

File: src/Routes/routes.js [hunk 1]
```diff
@@ -19,7 +20,11 @@ function routes(app) {
 
     app.get('/api/user/:id', jsonParser, UserController.getUserById, (req, res) => {
         res.send('Here is a specific user')
-    })
+    });
+
+    app.get('/api/track/:track', jsonParser, TrackController.getTrackResults, (req, res) => {
```
Comment (jdselby24) [inline src/Routes/routes.js]: Should it not be api/tracks/:track?

File: src/Services/validationService.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+/**
+ * Takes a http request and validates and sanitises data from body. Returns validated object of user data
+ *
+ * @param req HTTP Request
+ *
+ * @return object of validated user data
+ *
+ */
+function validateUserData (req) {
+
```
Comment (jdselby24) [inline src/Services/validationService.js]: Inconsistent whitespace

File: src/Routes/routes.js (hunk 1 above)
Comment (mgallagher56) [reply src/Routes/routes.js]: RESTful api says it should be singular, according to Mike

**Pull request merged**